cmd/openapi-gen/args/args.go

```diff
@@ -38,6 +39,8 @@ func NewDefaults() (*args.GeneratorArgs, *CustomArgs) {
 	// WithoutDefaultFlagParsing() disables implicit addition of command line flags and parsing,
 	// which allows registering custom arguments afterwards
 	genericArgs := args.Default().WithoutDefaultFlagParsing()
+	genericArgs.GoHeaderFilePath = filepath.Join(args.DefaultSourceTree(), "k8s.io/kube-openapi/boilerplate/boilerplate.go.txt")
```

Comment:
@apelisse I hit problem missing text file in gengo package: https://travis-ci.org/kubernetes/kube-openapi/builds/539024167. I suspect if the non-go files are ignored by go mod (v.s. previously `go get` persists the files unconditionally)

is there way to ping non-go files in dependency using go mod? Let me know what do you think

Reply:
What you did looks good I think ...